packages/inquirer-common/src/translations/inquirer-common.i18n.json

```diff
@@ -14,9 +14,6 @@
             "missingPropMsg": "The property: `{{destinationProperty}}` is missing.",
             "httpConnectionError": "A connection error occurred with the selected destination. Http code: {{- code}}. $t(texts.seeLogForDetails)"
         },
-        "cannotReadCapServiceMetadata": "An error occurred reading CAP service metadata: {{serviceName}}. $t(texts.seeLogForDetails).",
-        "capModelAndServicesLoadError": "An error occurred loading the CAP model and services. {{- error}}",
-        "capServiceUrlPathNotDefined": "An error occurred reading CAP service metadata: {{serviceName}}. CAP service property `urlPath` is not defined but is required.",
         "unknownError": "An error occurred{{- error, addMsgWithColonFormatter}}",
```

Comment:
Do you need a space after `...occurrred`?

Reply:
Its handled by the formatter: https://github.com/SAP/open-ux-tools/blob/main/packages/odata-service-inquirer/src/i18n.ts#L24